images/linux/scripts/installers/cassandra.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+################################################################################
+##  File:  homebrew.sh
+##  Desc:  Installs Apache Cassandra
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/os.sh
```

Comment:
I take it $HELPER_SCRIPTS is set somewhere else?

Reply:
Yes, that's correct. Those will get sourced as environment variables from other setup files. For e.g. [ubuntu2004.json has it here](https://github.com/actions/virtual-environments/blob/main/images/linux/ubuntu2004.json#L257).